src/common.jl

```diff
@@ -0,0 +1,28 @@
+
+function _load_embeddings_csv( embedding_file, max_vocab_size, keep_words)
```

Comment:
I think we can also use this for FastText maybe? there is at least some code overlap.
Might need to factor out the common inner parts.

We should probably not call it a CSV since it is space seperated,
Or we call it a CSV and pass in a deliminator option

Reply:
Deliminator option is added as suggested